index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Phychic Game</title>
+</head>
+<body>
+    <div>
+        <h3>Welcome to the Phychic Game</h3>
+        <h3>You Vs Computer, Who will WIN!!</h3>
+        <h3>Guess what letter I'm thinking of!</h3>
+        <p id=info-text>Please chose any letter between A-Z</p>
+        <p id=youchose-text></p>
+        <p id=computerchose-text></p>
+        <p id=guessesnumber-text></p>
+        <p id=win-text></p>
+        <p id=lose-text></p>
+        
+    </div>
+
+    <script type="text/javascript">
+
+    var computerChoices = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r",
+    "s", "t", "u", "v", "w", "x", "y", "z"];
+
+    var win = 0;
+    var lose = 0;
+    var guessesNumber = 9;
+    var youChose = [];
+
+    var infoText=document.getElementById("info-text");
+    var youChoseTest=document.getElementById("youchose-text");
+    var computerChoseText=document.getElementById("computerchose-text");
+    var guessesNumberText=document.getElementById("guessesnumber-text");
+    var winText=document.getElementById("win-text");
+    var loseText=document.getElementById("lose-text");
+    
+    document.onkeyup = function(event) {
+    var userGuess = event.key;
+    var computerGuess = computerChoices[Math.floor(Math.random() * computerChoices.length)];
+    var guess = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r",
+    "s", "t", "u", "v", "w", "x", "y", "z"];
+
+    if (guess.indexOf(userGuess.toLowerCase) === computerChoices) {
```

Comment:
I think you are suppose to just use the `userGuess` instead of this whole `guess.indexOf(userGuess.toLowerCase)` phrase since you already defined that the `userGuess` is the key you push and you want it to compare to `computerChoices`.

Reply:
Also is it suppose to be compared to the `computerGuess` instead of `computerChoices`?